examples/skr/maa-test/README.md

```diff
@@ -0,0 +1,103 @@
+# NOTE This example is for the Kata based AKS preview
+#      Today the supported GA solution for Kubernetes is to use virtual nodes on Azure Confidential Instances (https://learn.microsoft.com/en-us/azure/container-instances/container-instances-virtual-nodes)
+
+
+# Microsoft Secure Key Release (SKR) MAA Token Test on AKS Confidential Pods (i.e. based on Kata)
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Get an MAA Token Workflow Step By Step](#secret-provisioning-workflow)
+  - [1. Build the MAA test container](#1-build-the-maa-test-container)
+  - [2. Update the YAML file](#2-update-the-yaml-file)
+  - [3. Generate a Security Policy](#3-generate-a-security-policy)
+  - [4. Deploy](#4-deploy)
+
+## Introduction
+
+This guide provides instructions on how to use the Secure Key Release (SKR) sidecar to get an attestation token from a MAA endpoint.
+
+This requires the default HTTP endpoints in SKR. Therefore, there is no need to specify a port or any special environment variables, like in the [GRPC based example](../aks/skr-example-template.yaml).
+
+## Prerequisites
+
+- [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+    - `confcom` Azure CLI extension (run `az extension add -n confcom`)
+- `gettext` (for `envsubst`)
+- A container registry (Azure Container Registry, Docker Hub, etc.)
+- An AKS cluster set up with the Kata-based preview
+    - Follow instructions here to get started: https://learn.microsoft.com/en-us/azure/aks/deploy-confidential-containers-default-policy
+
+## Get an MAA Token Workflow Step By Step
+
+### 1. Build the MAA test container
+
+Review the contents of [Dockerfile.maa_test](../../../docker/skr/Dockerfile.maa_test) and update the attestation endpoint if you wish.
+
+Below are the MAA endpoints for the four regions in which Confidential Containers on AKS is currently available.
+
+- East US: sharedeus.eus.attest.azure.net
+- West US: sharedwus.wus.attest.azure.net
+- North Europe: sharedneu.neu.attest.azure.net
+- West Europe: sharedweu.weu.attest.azure.net
+
+Run Docker build on the files from the root directory, replacing <SKR_MAA_IMAGE> with the desired container name and version:
+
+```bash
+docker build -t <SKR_MAA_IMAGE> -f docker/skr/Dockerfile.maa_test .
+```
+
+Push the container images to your container registry.
+
+### 2. Update the YAML file
+
+If you don't have a registry yet, you can follow the instructions in the [ACI README](../aci/README.md) to set one up.
+
+Replace the image value <SKR_MAA_IMAGE> in the [SKR MAA Comms YAML](skr-maa-comms-test.yaml) with the full registry plus container name of your image, i.e. `registry-name.azurecr.io/skr-maa-comms:1.0`.
```

Comment:
Putting these two lines higher up before tagging and pushing the image would be cleaner